### ISSUE TITLE
Move the base image tarball build process to Docker

### ISSUE THREAD

**Issue by ezamriy** [AlmaLinux/docker-images#7]
Currently, I'm building the base tarball either on real hardware or on an AlmaLinux powered machine, but it would be really nice to move this process to Docker. That will allow us to automate things easier.

I'm pretty sure it should work if we create a Docker image with systemd enabled.

**Comment by srbala** [comment]
We have the https://github.com/AlmaLinux/ks2rootfs container utility ready, intetrating with `CI/CD` process would next step.

**Comment by srbala** [comment]
Since `almalinux/ks2rootfs` is available on docker hub, anyone can do docker development in any environment as long as `docker` or `podman` is available. Commands below `docker` can be replaced with `podman`. The `micro` and `init` PR #14 fully developed using this approach.

New issue will be created for `workflow` automation using `CI/CD` process

### creating default or minimal
```sh
docker run --rm --privileged -v "$PWD:/build:z" \
    -e BUILD_KICKSTART=kickstarts/almalinux-8-default.x86_64.ks \
    -e BUILD_ROOTFS=almalinux-8-default-docker.x86_64.tar.gz \
    -e BUILD_OUTDIR=work/default84 \
    almalinux/ks2rootfs
```

```sh
docker run --rm --privileged -v "$PWD:/build:z" \
    -e BUILD_KICKSTART=kickstarts/almalinux-8-minimal.x86_64.ks \
    -e BUILD_ROOTFS=almalinux-8-minimal-docker.x86_64.tar.gz \
    -e BUILD_OUTDIR=work/minimal84 \
    almalinux/ks2rootfs
```
These two below are ready to build after #14 merge.
```sh
docker run --rm --privileged -v "$PWD:/build:z" \
    -e BUILD_KICKSTART=kickstarts/almalinux-8-micro.x86_64.ks \
    -e BUILD_ROOTFS=almalinux-8-micro-docker.x86_64.tar.gz \
    -e BUILD_OUTDIR=work/micro84 \
    almalinux/ks2rootfs
```

```sh
docker run --rm --privileged -v "$PWD:/build:z" \
    -e BUILD_KICKSTART=kickstarts/almalinux-8-init.x86_64.ks \
    -e BUILD_ROOTFS=almalinux-8-init-docker.x86_64.tar.gz \
    -e BUILD_OUTDIR=work/init84 \
    almalinux/ks2rootfs
```
